internal/cmd/osac-dev/generate/openapi/generate_openapi_cmd.go

```
@@ -0,0 +1,504 @@
+	// Calculate the absolute path of the cacheFile in the cache:
+	cacheFile := filepath.Join(cacheDir, args.Name)
+	cacheFile, err = filepath.Abs(cacheFile)
+	if err != nil {
+		return
+	}
+
+	// Do nothing if the file already exists:
+	_, err = os.Stat(cacheFile)
+	if err == nil {
+		result = cacheFile
+		return
```

golangci-lint (2.12.2)

[error] error is nil (line 402) but it returns error

(nilerr)